src/StoreKeeper/WooCommerce/B2C/Tools/OrderHandler.php

```diff
@@ -33,6 +39,10 @@ public function updateWithIgnore($order_id): void
     {
         global $storekeeper_ignore_order_id;
 
+        if (!$this->isSyncAllowed($order_id)) {
```

Comment:
manual sync still should be allowed though, how you made it now the button will say success but nothing will happen

Reply:
This won't affect manual sync as `OrderHandler` only handles those woocommerce hooks.
I added a note on metabox when order sync is disabled.